packages/monitors/test/index.js

```diff
@@ -155,24 +157,23 @@ contract("index.js", function(accounts) {
       toBlock,
       defaultMonitorConfig,
       defaultUniswapPricefeedConfig,
-      defaultMedianizerPricefeedConfig
+      defaultUniswapPricefeedConfig
     );
 
-    // Iterate over all log events and count the number of empStateUpdates, liquidator check for liquidation events
+    // Iterate over all log events and count the number of tokenBalanceStorage, liquidator check for liquidation events
     // execution loop errors and finally liquidator polling errors.
     let reTryCounts = {
-      empStateUpdates: 0,
+      tokenBalanceStorage: 0,
```

Comment:
This is a result of the main `Promise.all` refactor in the `index.js` of the bot. Previously the bot would update the state in a different order. First, it would update some clients, then run some monitor modules, then update more clients then run more modules. After this PR all clients are updated in parallel then all modules are run in parallel.

Due to this re-ordering the log message for `Expiring multi party event state updated` is not printed before the bot errors out and re-tries it's next loop. However, the message for updating bot balances is :)

it' a bit roundabout and confusing but this was the only way I could find to consistently test the re-try logic.

Reply:
SGTM!